src/com/itacademy/eshop/product/Books.java

```diff
@@ -0,0 +1,9 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.product.types.Category;
+
+public abstract class Books extends Product implements ReadableOnline{

```

Comment:
Empty abstract class. This class serves no purpose, Check difference between your version and Electronics -> Computer. In this case you could just drop this Book class an directly extend Stories with Product